MIGRATION.md

```diff
@@ -864,6 +865,43 @@ import type { SvelteRenderer } from '@storybook/svelte';
 // etc.
 ```
 
+#### Renamed `DecoratorFn` to `Decorator`
```

Comment:
I also feel that we should mention that this change impacts all frameworks. All examples are react, maybe people will get confused?

Reply:
We didn’t export `DecoratorFn` in other frameworks in 6.x I believe.